NeoSwift/Util/Data+Hex.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Data+Hex.swift
+//  NeoSwift
+//
+//  Created by Luís Silva on 13/09/17.
+//  Copyright © 2017 drei. All rights reserved.
+//
+
+import Foundation
+
+extension Data {
```

Comment:
Sweet definitely a handy extension, and good to keep them in here rather than an external dependency

Reply:
I had to introduce the `bytes` property to the Data extension, so now it includes the hex string functionality and the converter to [UInt8] `bytes` property, and I renamed it Data+Util. Do you think it might be better to separate the `bytes` property and hex string properties into different files?